entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -130,7 +114,7 @@ class EntityService(
         entity: Entity,
         propertyKey: String,
         propertyValues: Map<String, List<Any>>

```

Comment:
Could we have an ExpandedProperty or ExpandedAttribute to have the same utils method as for ExpandedEntity and get relationships/properties directly ?

Reply:
Yes, having this kind of abstraction for properties and relationships is planned for a near future